app/components/post_list/post/avatar/avatar.tsx

```diff
@@ -112,8 +115,8 @@ const Avatar = ({author, enablePostIconOverride, isAutoReponse, location, post}:
             author={author}
             size={ViewConstant.PROFILE_PICTURE_SIZE}
             iconSize={24}
-            showStatus={!isAutoReponse || author.isBot}
-            testID={`post_avatar.${author.id}.profile_picture`}
+            showStatus={!isAutoReponse || author?.isBot}
+            testID={`post_avatar.${author?.id}.profile_picture`}
```

Comment:
Since we're returning early in this function, these question marks aren't needed. This stood out to me, as we definitely don't want `undefined` in the `testID`, but that won't ever happen here anyway.

Reply:
The early return is actually in a different function, so this may happen.

That being said, this should not happen, since we should always have an author in the database, but this is a way to make sure the app doesn't crash.